src/test.ts

```
@@ -0,0 +1,154 @@
+            sender: '00',
+            recipient: '555dc5d04e4c11e89b44174d1b876bbf',
+            transactionId: 'bcf84b704e4c11e89b44174d1b876bbf',
+        },
+    ],
+    currentNodeUrl: 'http://localhost:3001',
+    networkNodes: [],
+}
+
+console.log('VALID: ', bitcoin.chainIsValid(bc1.chain).isOk)
+console.log(bitcoin.chainIsValid(bc1.chain).isOkAnd(() => true))
+class Number implements Default<Number> {
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)